package.json

```diff
@@ -54,5 +54,7 @@
     "sinon": "^7.4.1",
     "typescript": "^3.2.4"
   },
-  "dependencies": {}
+  "dependencies": {
+    "@types/uuid": "^7.0.2"
```

Comment:
types should go to `devDependencies`, also i see uuid dependency is missing in package.json, we need to include it under `dependencies` if we rely on it.

Reply:
✅ Done :)